packages/launch-editor/guess.js

```diff
@@ -76,6 +76,11 @@ module.exports = function guessEditor (specifiedEditor) {
     // Ignore...
   }
 
+  // Check if user chose the LAUNCH_EDITOR escape hatch
+  if (process.env.LAUNCH_EDITOR) {
```

Comment:
should this be done at the beginning of "guess" or even outside of it, it isn't a guess if the user specified an explicit command to use. 

In fact if a different editor was found in the process list it would still be preferred over LAUNCH_EDITOR in the current implementation

(sorry for not asking this sooner)

Reply:
I made it like this to have the changes be as low profile as possible. This way it would definitely not disturb how launch-editor operated before and be non-breaking.
I guess the amount of people who use this tool already and happen to have a LAUNCH_EDITOR environment variable set is pretty negligible though...?

Reply:
Technically it would be a breaking change either way for people that already have the environment variable set. a google search for "LAUNCH_EDITOR" did turn up very few results (<3k) and skimming them i didn't find documented use of it.

For extra safety, `LAUNCH_EDITOR_COMMAND` has 3 results and all of them seem to point at one python lib config option from 2018.